### ISSUE TITLE
BUG: MAL is not syncing correctly. 

### ISSUE THREAD

**Issue by Droyk** [tofuness/Toshocat#10]
You are about to read some bad English. and I am really sorry about it. I am not a native English speaker.
I am not really good at explaining things so plz forgive if you don't understand a thing.  

Hey guys I just found a Bug So When I first Installed Toshocat and sync my anime list
It got synced correctly But there is a bug in it 

Toshocat only syncs correctly from the MAL for the first time only after that it only updates MAL list.
But if the user add a anime through MAL not from toshocat it won't sync a damn thing 

I tried too many times I even reload toshocat and re add my credentials but it won't sync MAL it just update MAL. Is there is anything I can do? 

Edit: I forgot to write that the syncing message comes up and it even shows it got synced correctly but in reality it won't sync a damn thing.

**Comment by tofuness** [comment]
It takes a few minutes to update.

Toshocat gets its info directly from MyAnimeList and it takes a few minutes for this endpoint to show the changes: http://myanimelist.net/malappinfo.php?type=anime&status=all&u=USERNAME

**Comment by Droyk** [comment]
How many minutes do I have to wait? because it's been 10 minutes but it didn't sync at all. 
I can only see that blue popup that tell's you it's syncing then after 1 to 2 seconds the blue popup change into green and says syncing complete but still in reality it didn't at all.  

why does it take that much to being with?

**Comment by tofuness** [comment]
@Droyk I'll look into it!

**Comment by Droyk** [comment]
@tofuness Why does it take that much to being with?

**Comment by tofuness** [comment]
@Droyk The MyAnimeList API doesn't update instantly - its results are cached. I'll try and contact MyAnimeList and see if they can fix it on their side.

**Comment by Droyk** [comment]
Okay by the way how much it took it on your system? 
Well For me its been 25 minutes since I open Toshocat but still no syncing :(

**Comment by Droyk** [comment]
One hour still no progress.. can you plz tell where you store the synced Anime list. I am thinking of deleting the list maybe that can help?

**Comment by tofuness** [comment]
@Droyk
1. Open up the developer tools through the top-left menu.
2. Go to the _resources_ tab > click on _local storage_ > _file://_.
3. Finally, select the `list.myanimelist` line and delete it (and then reload the app).

![untitled-1](https://cloud.githubusercontent.com/assets/2232877/17303289/4e4e3000-5820-11e6-9bfa-63a2cf36fa93.png)

**Comment by Droyk** [comment]
Okay that fixed it but not completely but at least now I have a workaround until you patch this bug up. 
And I just notice something Every time I hit Re-sync with MyAnimeList on settings panel it give the below error on developer console. See if this helps you to solve the problem 
![screenshot_20160801231149](https://cloud.githubusercontent.com/assets/15848350/17303477/7d343b4e-583e-11e6-8bdb-8b49a2a7df62.png)

**Comment by Droyk** [comment]
Hey what happened @tofuness did you stop developing this anymore?

**Comment by tofuness** [comment]
@Droyk 
Development currently stalled. Got other things to do atm.

**Comment by Droyk** [comment]
So do you plan to work on this again if yes will you be able to tell me an approximate time it's going to take?

**Comment by tofuness** [comment]
@Droyk Yes, I do plan on working on this again. Hopefully I'll get a new major version out on a month or two!

**Comment by Droyk** [comment]
Okay thanx :)
